static/app/views/monitors/components/overviewTimeline/index.tsx

```diff
@@ -95,6 +95,22 @@ export function OverviewTimeline({monitorList}: Props) {
     });
   };
 
+  const handleToggleStatus = async (monitor: Monitor) => {
+    const data: Partial<Monitor> = {
+      status: monitor.status === 'active' ? 'disabled' : 'active',
+    };
+
+    const resp = await updateMonitor(api, organization.slug, monitor.slug, data);
```

Comment:
Small bug here. When we update we don't pass along the environments filter, so updates come back with ALL environments, which isn't what we actually want.

Reply:
hmm what happens with this bug, does that mean that all environments will be shown after disabling even if you have only a few selected?

Reply:
yea it might be a little bit more annoying but it's probably better